lib/pharos/addons/cert_manager.rb

```diff
@@ -9,7 +9,7 @@ class CertManager < Pharos::Addon
 
       class Issuer < Pharos::Addons::Struct
         attribute :name, Pharos::Types::String
-        attribute :server, Pharos::Types::String
+        attribute :server, Pharos::Types::String.optional
```

Comment:
Unrelated change?

Reply:
Yes / No. It's optional in the schema below, so it's a bug on it's own, but with the namespace thing it makes it explode unless it's filled.

Reply:
Right, these need stricter validation now... erb templates defaulted typo'd variables to `nil`.

Reply:
BTW strong :+1: for this change, because the current behavior means that typos in erb templates or changes to `apply_stack` options can silently break... something like the following would end up deploying weave without encryption, which would suck hard:

```patch
diff --git a/lib/pharos/resources/weave/daemon-set.yml b/lib/pharos/resources/weave/daemon-set.yml
index 50f6c52..8a18032 100644
--- a/lib/pharos/resources/weave/daemon-set.yml
+++ b/lib/pharos/resources/weave/daemon-set.yml
@@ -27,7 +27,7 @@ spec:
               value: "0"
             - name: CHECKPOINT_DISABLE
               value: "1"
-            <% if !trusted_subnets.empty? %>
+            <% if !trusted_sbunets.empty? %>
             - name: EXTRA_ARGS
               value: "--trusted-subnets <%= trusted_subnets.join(',') %>"
             <% end %>
```